### ISSUE TITLE
Relaxation scheme variable product

### ISSUE THREAD

**Issue by hakanergun** [lanl-ansi/InfrastructureModels.jl#53]
Could we add a similar function as below, for not rotated second order cones as variable d is not defined in these cases. An example could be:

"constraint: `c^2 <= a*b`"
function relaxation_complex_product_conic(m::JuMP.Model, a::JuMP.VariableRef, b::JuMP.VariableRef, c::JuMP.VariableRef)
    a_lb, a_ub = InfrastructureModels.variable_domain(a)
    b_lb, b_ub = InfrastructureModels.variable_domain(b)

    @assert (a_lb >= 0 && b_lb >= 0) || (a_ub <= 0 && b_ub <= 0)

    JuMP.@constraint(m, [a/sqrt(2), b/sqrt(2), c, 0] in JuMP.RotatedSecondOrderCone())
end

Or maybe define d as a keyword argument and if not provided, set to zero by default in the original function?

**Comment by ccoffrin** [comment]
@hakanergun sure can you make the PR?  The form would be,
```
JuMP.@constraint(m, [a/sqrt(2), b/sqrt(2), c] in JuMP.RotatedSecondOrderCone())
```
An overload of `relaxation_complex_product_conic` should be fine given that the signature is distinct.

**Comment by ccoffrin** [comment]
Resolved by #55.
